scripts/train/train.py

```diff
@@ -248,7 +248,8 @@ def main(cfg: DictConfig) -> Trainer:
                                                must_exist=True)
     eval_interval: Union[int, str] = pop_config(cfg,
                                                 'eval_interval',
-                                                must_exist=True)
+                                                default_value=1,
```

Comment:
Does this default to 'ba', if so 1 might be too high?

Reply:
I just wanted to match the composer default. I think `1` is interpreted as `1ep`. (And if eval_loader is not specified, then this has no effect)